packages/deeplinks/analytics/deep_links.ts

```diff
@@ -8,6 +8,12 @@
 
 import { DASHBOARD_APP_ID, DISCOVER_APP_ID, VISUALIZE_APP_ID } from './constants';
 
-export type AppId = typeof DISCOVER_APP_ID | typeof DASHBOARD_APP_ID | typeof VISUALIZE_APP_ID;
+export type DiscoverApp = typeof DISCOVER_APP_ID;
+export type DashboardApp = typeof DASHBOARD_APP_ID;
+export type VisualizeApp = typeof VISUALIZE_APP_ID;
 
-export type DeepLinkId = AppId;
+export type AppId = DiscoverApp | DashboardApp | VisualizeApp;
+
+export type DiscoverAppProfile = 'log-explorer';
```

Comment:
Based on the work done for [using the deep link ids instead of the href](https://github.com/elastic/kibana/pull/159125), setting this type as part of the DeepLinkId union will allow accessing this deep link when it'll be referenced on the Observability serverless project navigation sidebar.

Reply:
Ah, I see. Would it make sense to move it to the observability deep links package, then?

Reply:
I can do it, I added this in the analytics package since the whole definition for discover was there, but make sense to have it in the obs package.